modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/RichTextField.testcase

```diff
@@ -220,7 +220,33 @@ definition {
 	test SetFieldReferenceOfRichTextWithInvalidCharacters {
 		property portal.acceptance = "false";
 
-		// TODO LPS-98076 SetFieldReferenceOfRichTextWithInvalidCharacters pending implementation
+		DataEngine.addField(
+			fieldFieldLabel = "Test Invalid Reference",
+			fieldName = "Rich Text"
+		);
+		
+		AssertClick.assertPartialTextClickAt(
+			key_fieldFieldLabel = "Test Invalid Reference",
+			locator1 = "DDMEditStructure#FORM_FIELD_CONTAINER",
+			value1 = "Test Invalid Reference"
+		);
+		
+		Navigator.gotoNavTab(navTab = "Advanced");
+
+		Type(
+			key_fieldLabel = "Field Reference",
+			key_fieldName = "fieldReference",
+			locator1 = "Sidebar#TEXT_INPUT",
+			value1 = "Test!Invalid@#Reference"
+		);
+
+		AssertTextNotPresent(
```

Comment:
Hey @nicolas-sss , instead of validate that value1 is not present, could you please validate that the expected value is present? e.g.: TestInvalidReference 

Validating this way we will make sure the right value is being set. The current assert is only validating the not expected text is not there, but any other value could be.